api/go/api_polls_service.go

```diff
@@ -241,6 +241,15 @@ func (s *PollsApiService) UpdatePoll(ctx context.Context, xUSERID string, pollID
 		return Response(http.StatusBadRequest, poll_model)
 	}
 
+	// After the update is successful, delete all of the votes
+	_, deleteErr := firestore_client.Collection(votesCollectionName).Doc(pollID).Delete(ctx)
+
+	if deleteErr != nil {
+		AddMessage(&messages, Severity(ERROR), "Request Body issue", fmt.Sprintf("Poll votes could not be deleted but the Update was successful: %s", err))
+		poll_model.Messages = messages
+		return Response(http.StatusBadRequest, poll_model)
+	}
```

Comment:
How do you feel about adding a warning modal when you click the update button letting the user know that the votes will be cleared? Can be a post mvp thing and not related to backend